WTWSMS/common/religions/00_religions.txt

```diff
@@ -2516,12 +2558,13 @@ pagan_group = {
 		female_temple_holders = yes
 		feminist = yes
 		defensive_attrition = yes
-		max_consorts = 3
 		peace_prestige_loss = yes # Lose Prestige while at peace
 		raised_vassal_opinion_loss = no
 		allow_looting = yes
 		reformed = celtic_pagan_reformed
 		aggression = 1.5 # AI aggression factor
+		bs_marriage = yes
+		pc_marriage = yes
```

Comment:
This whole attitude may have been the faults of the Romans, who had very relaxed views on incest (Caligula married his sister and declared her his Goddess sister-wife). It was not actually a crime, but was often used in a slanderous manner.

I can confirm neither of the four men above were slandered for their choice of partner (quasi-nuclear relatives) but only by their other actions. Gildas casually mentions Vortipor's incest then says he's evil since he usurped the throne.